src/script/app.js

```diff
@@ -0,0 +1,38 @@
+// Hidden navbar
+let lastKnownScrollPosition = 0;
+
+document.addEventListener("scroll", function () {
+  if (window.scrollY > lastKnownScrollPosition) {
+    document.querySelector(".nav").classList.add("nav_hidden");
+  } else {
+    document.querySelector(".nav").classList.remove("nav_hidden");
+  }
+  lastKnownScrollPosition = window.scrollY;
+});
```

Comment:
Read about the `throttle` function, also open the JavaScript console and check how many times this code executes.